vuepress/docs/next/docs/concepts/operator-intro.md

```diff
@@ -0,0 +1,64 @@
+# The Entando Operator
+
+The Entando Operator processes the custom resources in Kubernetes that represent the different [components of an Entando application](./README.md).
+The goal of the operator is to provide automation and a set of default configuration to simplify and accelerate the deployment of an Entando application.
```

Comment:
Maybe "set of default configuration options"?

Reply:
Done